README.md

```diff
@@ -16,9 +16,9 @@ Recommended Qt framework of version 5.9.1 LTS.
 
 ## Getting Started
 
-- [Install Prerequisites](docs/ReactQt/InstallPrerequisites.md)  
-- [Install adjusted react-native-cli](docs/ReactQt/InstallUpdatedReactNativeCLI.md)  
-- [Create new App](docs/ReactQt/CreateNewApp.md)  
+- [Install Prerequisites](docs/InstallPrerequisites.md)
```

Comment:
I thought I had done this in #271. Sorry for the confusion. 😕

Reply:
Can you fix all of the links to the docs? `docs/ReactQt/*` -> `docs/*`

Reply:
no worries! super small change. and good call, just pushed the rest.